packages/web-extension/src/walletManager/walletManager.types.ts

```diff
@@ -46,11 +53,28 @@ export interface WalletManagerActivateProps<P extends string | number = string,
 }
 
 export interface WalletManagerApi {
+  chainId$: Observable<Cardano.ChainId>;
+
   /**
    * Create and activate a new ObservableWallet.
    * Reuses the store if the wallet was previously deactivated but not destroyed.
    */
   activate(props: WalletManagerActivateProps): Promise<void>;
+
+  /**
+   * Returns true if the wallet is active.
+   *
+   * @param props The properties of the wallet to check.
+   */
+  isActive(props: WalletManagerActivateProps): Promise<boolean>;
```

Comment:
This suggestion is only valid if we keep activation functionality to set walletId only
```suggestion
  isActive(walletId: WalletId): Promise<boolean>;
```

Reply:
Maybe we should delete this method? Doesn't seem very useful since we add the observable for active wallet props

Reply:
agree will remove

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1012/commits/cd24430f838afcaaa5d8271ec804d2f53a167ad4